docs_outdoornav_user_manual/integration_requirements/installation_instructions.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Installation Instructions"
+sidebar_label: "Installation Instructions"

```

Comment:
Suggest we call this "Software Integration Instructions" since Appendix A is a mixture of both hardware and installation. Adding "Software" will clarify the purpose of this page when scanning through on the sidebar.